node/expressjs/middlewares-in-express-js.md

```diff
@@ -0,0 +1,61 @@
+## Middlewares in Express JS
+
+It's been a long time that NODE JS has become a part of JS Ecosystem. Any JS developer Front End or Full Stack would be familiar with Node JS. Any one who has worked on Node would have also worked with Express JS.
```

Comment:
```suggestion
It has been a long time since Node.js was released. It has now become an integral part of the JS ecosystem. Any JS developer, be it Frontend or Fullstack would probably be familiar with Node.js. Anyone who has worked on Node would have also worked with Express JS.
```